src/libs/DistanceRequestUtils.js

```diff
@@ -102,12 +102,12 @@ const getDistanceMerchant = (hasRoute, distanceInMeters, unit, rate, currency, t
  * @param {Number} distance - The distance traveled in meters
  * @param {'mi' | 'km'} unit - The unit of measurement for the distance
  * @param {Number} rate - Rate used for calculating the request amount
- * @returns {Number} The computed request amount.
+ * @returns {Number} The computed request amount in "cents" (integer).
```

Comment:
NAB - do you like this better? 

```suggestion
 * @returns {Number} The computed request amount (rounded) in "cents".
```